codejail/safe_exec.py

```diff
@@ -80,8 +86,22 @@ def safe_exec(
     the code raises an exception, this function will raise `SafeExecException`
     with the stderr of the sandbox process, which usually includes the original
     exception message and traceback.
-
     """
+    if ALWAYS_BE_UNSAFE:
+        not_safe_exec(
+            code,
+            globals_dict,
+            files=files,
+            python_path=python_path,
+            limit_overrides_context=limit_overrides_context,
+            slug=slug,
+            extra_files=extra_files,
+        )
+        return
+
+    if not jail_code.is_configured('python'):
+        raise RuntimeError("safe_exec has not been configured for Python")
```

Comment:
Can we add a test for this?

Reply:
Good catch. Tests added.

This sent me down a rabbit hole for "how the heck do tests actually work" and I ended up adding some docs. This also cleared up some confusion about how codejail works in production, actually...